core/src/main/java/org/opensearch/sql/expression/function/FunctionResolver.java

```diff
@@ -1,68 +1,13 @@
-/*
```

Comment:
Removing copyright?

Reply:
Addressed in https://github.com/Bit-Quill/opensearch-project-sql/pull/99/commits/277ee6ad9485d4a0ad3097cec7af916894e14284